packages/image_picker/image_picker_platform_interface/lib/src/method_channel/method_channel_image_picker.dart

```diff
@@ -72,13 +73,18 @@ class MethodChannelImagePicker extends ImagePickerPlatform {
       throw ArgumentError.value(maxHeight, 'maxHeight', 'cannot be negative');
     }
 
+    if (limit != null && limit < 2) {
+      throw ArgumentError.value(limit, 'limit', 'cannot be lower than 2');
+    }
```

Comment:
This is an Android requirement, iOS doesn't have this kind of limitation. Although there's a little difference between calling it with 1 and 2, because if you call iOS picker with limit equal or more than 2, then you have to confirm your selection with the 'add' button. And if you call iOS picker with limit = 1, then picker takes the first selected photo without confirmation. 
I thought it would be better to set such a requirement everywhere, rather than doing a different behaviour for Android.

```
@RequiresApi(19)
open class PickMultipleVisualMedia(
    private val maxItems: Int = getMaxItems()
) : ActivityResultContract<PickVisualMediaRequest, List<@JvmSuppressWildcards Uri>>() {

    init {
        require(maxItems > 1) {
            "Max items must be higher than 1"
        }
    }
```